tor_core/config.py

```diff
@@ -156,6 +156,9 @@ class Config:
 
 # ----- Compatibility -----
 config = Config
+config.name = None
+config.bot_version = 0  # this should get overwritten by the bot process
```

Comment:
Because this is in the core, it just gets overwritten with whatever gets passed in, so type doesn't matter. That being said, it would probably be clearer if I replaced it with `'0.0.0`.